### PR TITLE
docs: Add examples for all the adi devices

### PR DESCRIPTION
**Pull request by zabackary**
## Describe the changes this PR makes. Why should it be merged?

I had some time and did the not-fun work of adding examples. Hopefully they compile; it doesn't seem like there's a way to `cargo check` doctests yet.

## Additional Context

- These are *only* non-code changes (e.g. documentation, README.md).

### CODE REVIEW

**Review by Gavin-Niederman — APPROVED** [review]
LGTM otherwise. Ty :3

**Review by Gavin-Niederman — CHANGES_REQUESTED** [review]
Chose wrong type

**Comment by zabackary** [comment]
> Chose wrong type

Wait, for what?

**Comment by Gavin-Niederman** [comment]
> > Chose wrong type
> 
> Wait, for what?

I clicked the wrong review type and accidentally approved.

File: packages/vexide-devices/src/adi/light_sensor.rs [hunk 1]
```diff
@@ -43,22 +58,37 @@ impl AdiLightSensor {
...
         Ok(f64::from(analog::ADC_MAX_VALUE - self.raw_brightness()?)
             / f64::from(analog::ADC_MAX_VALUE))
     }
 
     /// Returns the 12-bit brightness reading of the sensor.
     ///
-    /// This is a raw 12-bit value from [0, 4095] representing the voltage level from
-    /// 0-%V measured by the V5 Brain's ADC.
+    /// This is a raw 12-bit value in the interval [0, 4095] representing the voltage level from
+    /// 0-5V measured by the V5 Brain's ADC.
```
[the remark this answers is not in the document]
Comment (Gavin-Niederman) [reply packages/vexide-devices/src/adi/light_sensor.rs:91]: Yes I think what you have is correct.

**Review by Gavin-Niederman — APPROVED** [review]
LGTM TYSM :3

**Pull request merged**